tests/TestUtils.hs

```diff
@@ -165,3 +165,8 @@ pushd dir act = do
   result <- try act :: IO (Either IOException a)
   setCurrentDirectory original
   either throwIO return result
+
+generateResolutionsFile :: FilePath -> IO ()
+generateResolutionsFile fp =
+  readProcess "bower" ["list", "--offline", "--json"] "" >>=
```

Comment:
I don't think so - I would expect `readProcess` to produce data incrementally, and `writeFile` should consume its input string entirely, so the resource should be cleaned up when the input string is fully consumed.